src/application/SoundManager.java

```diff
@@ -55,27 +70,39 @@ private void mp3_player(String soundFilePath){
         }
     }
 
-    private void wav_player(String soundFilePath){
+    private void player_and_thread_stop() {
```

Comment:
kedze ju treba zavolat aj v inych triedach tak ju daj public a premenuj ju a aj sound_time pomocou CamelCasu nech to 
 v class diagrame nejako vyzera, kedze vsetko mame CamelCasom